src/QsCompiler/LanguageServer/LanguageServer.cs

```diff
@@ -185,6 +185,12 @@ internal async Task CheckDotNetSdkVersionAsync()
         /// </summary>
         internal void OnInternalError(Exception ex)
         {
+            var telemetryProps = new Dictionary<string, string?>();
```

Comment:
Thanks for finding this! I assumed we were capturing all errors/exceptions in the previous telemetry implementation!

Reply:
No, this was long overdue. It would be great if we could actually have some tracking for this number, and raise some flags if the number goes up.

Reply:
> ...and raise some flags if the number goes up.

We currently have similar telemetry on the VS Code extension and IQ#, but no flags if they go up; perhaps it would be good to include those as well?

Reply:
I'll create a work item for us to add monitoring and alerts for these events.